packages/concerto-cto/lib/parser.pegjs

```diff
@@ -73,10 +73,10 @@
     }
     const start = value.start;
     const end = value.end;
-    start.$class = 'concerto.metamodel@1.0.0.Position';
-    end.$class = 'concerto.metamodel@1.0.0.Position';
+    start.$class = 'concerto.metamodel@1.1.0.Position';
```

Comment:
These version number changes are making the diff messy. Can we take the hit to move the version number into a variable of some sort in the parser so that in future all the rules don't change when we increment the version number?